### DISCUSSION TITLE
debug & future of planify

### DISCUSSION THREAD

**Discussion by HDInfautre** [alainm23/planify#1064] [General]
Hello
I would like to remind you once again.
I immediately appreciated and adopted your application, because I find the ergonomics to be really great.
However, after transferring a large part of my tasks from the everdo application, I encountered several bugs which were noted by you.
Despite my reminders, I have not received any feedback or a forecast date.
It seems that this project is done by a single person on a voluntary basis.
As is often the case, I understand that donations are not enough and that it may be necessary to complete on support or an inexpensive paid version, but which allows us to respond to the most important bugs and feature requests.
Today I started using Todoist for my planned tasks but even though the ergonomics are excellent, I don't like having my data stored outside of my computer.
I will need to know if you will soon take into account the most urgent requests or perhaps, you have personal reasons to put this project on hold.
https://github.com/alainm23/planify/issues/1057
https://github.com/alainm23/planify/issues/1037
https://github.com/alainm23/planify/issues/1032
and
https://github.com/alainm23/planify/issues/1033
https://github.com/alainm23/planify/issues/1034
Please answer me so that I can decide how to better manage my tasks.
Regards

**Comment by alainm23** [comment]
Thank you for using Planify, I have returned to active development and have closed most of your issues. Planify is still under development and new features are constantly being added, in December alone we have already done 3 releases. I will close this discussion and don't stop submitting issues, it's the best way to keep Planify growing.
